R/tar_quarto_files.R

```diff
@@ -79,12 +98,28 @@ tar_quarto_files_project <- function(path) {
       "quarto.org to learn how to set output-dir in _quarto.yml."
     )
   )
+
+  # Detect source files.
+  sources <- info$files$input[file.exists(info$files$input)]
+
+  # Detect input files.
   input <- info$files
-  input$input <- NULL
   input <- unlist(input)
   input <- input[file.exists(input)]
+  for (myfile in names(info$fileInformation)) {
+    input <- c(
+      input,
+      # `myfile` is an absolute path.
+      myfile,
+      # `includeMap` files are relative starting from `myfile`.
+      file.path(dirname(myfile), info$fileInformation[[myfile]]$includeMap$target)
```

Comment:
Would you break this line up into separate lines?

Reply:
:+1: